pkg/cmd/server/start/start_master.go

```diff
@@ -533,6 +534,16 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 	oc.RunServiceAccountPullSecretsControllers()
 	oc.RunSecurityAllocationController()
 
+	// TODO(miminar): run origin resource quota controller by default after 3.2
+	// run origin resourcequota controller only if the quota admission plugin is enabled
+	runOriginResourceQuotaController := false
```

Comment:
I would not have expected this to be the way we disable the quota controller for origin.  Is this because we have no single config flag to look for?